src/integration-test/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectReportControllerIntegrationTest.java

```diff
@@ -85,6 +85,7 @@ public void setup() throws IOException {
                .queryParam("name", bucketObjectName)
                .queryParam("commitMessage", "commit message")
                .queryParam("objectContentType", MediaType.APPLICATION_XML.toString())
+               .queryParam("projectName", MediaType.APPLICATION_XML.toString())
```

Comment:
MediaType.APPLICATION_XML.toString() is not the default value (or example usage) of projectName